src/routes/safe/components/Transactions/helpers/TxParametersDetail/index.tsx

```diff
@@ -99,43 +126,146 @@ export const TxParametersDetail = ({
-            <ColoredText
-              size="lg"
-              isOutOfOrder={isTxNonceOutOfOrder}
-              color={areSafeParamsEnabled(parametersStatus || defaultParameterStatus) ? 'text' : 'secondaryLight'}
-            >
+            <ColoredText size="lg" isOutOfOrder={isTxNonceOutOfOrder} color={getColor()}>
               Safe nonce
             </ColoredText>
-            <ColoredText
-              size="lg"
-              isOutOfOrder={isTxNonceOutOfOrder}
-              color={areSafeParamsEnabled(parametersStatus || defaultParameterStatus) ? 'text' : 'secondaryLight'}
-            >
+            <ColoredText size="lg" isOutOfOrder={isTxNonceOutOfOrder} color={getColor()}>
               {txParameters.safeNonce}
             </ColoredText>
           </TxParameterWrapper>
 
           {showSafeTxGas && (
             <TxParameterWrapper>
-              <Text
-                size="lg"
-                color={areSafeParamsEnabled(parametersStatus || defaultParameterStatus) ? 'text' : 'secondaryLight'}
-              >
+              <Text size="lg" color={getColor()}>
                 SafeTxGas
               </Text>
-              <Text
-                size="lg"
-                color={areSafeParamsEnabled(parametersStatus || defaultParameterStatus) ? 'text' : 'secondaryLight'}
-              >
+              <Text size="lg" color={getColor()}>
                 {txParameters.safeTxGas}
               </Text>
             </TxParameterWrapper>
           )}
           <StyledButtonLink color="primary" textSize="xl" onClick={onEdit}>
             Edit
           </StyledButtonLink>
+          {storedTx?.length > 0 && <TxAdvancedParametersDetail tx={storedTx[0]} />}
         </AccordionDetailsWrapper>
       </AccordionDetails>
     </Accordion>
   )
 }
+
+const TxParameterEndWrapper = styled.span`
+  display: flex;
+  justify-content: flex-end;
+  gap: 4px; // EthHashInfo uses a gap between the address and copy button
+`
+
+const TxAdvancedParametersDetail = ({ tx }: { tx: Transaction }) => {
+  const { txData, detailedExecutionInfo } = tx?.txDetails || {}
+
+  if (!txData || !detailedExecutionInfo) {
+    return null
+  }
+
+  const { value, to, operation, hexData } = txData
+  const { safeTxHash, baseGas, gasPrice, gasToken, refundReceiver, confirmations } =
+    (isMultiSigExecutionDetails(detailedExecutionInfo) && detailedExecutionInfo) || {}
+
+  return (
+    <>
+      <StyledDivider />
+
+      {value && (
+        <TxParameterWrapper>
+          <Text size="lg">value</Text>
+          <Text size="lg">{value}</Text>
+        </TxParameterWrapper>
+      )}
+
+      {to?.value && (
+        <TxParameterWrapper>
+          <Text size="lg">to</Text>
+          <PrefixedEthHashInfo
+            textSize="lg"
+            hash={to.value}
+            showCopyBtn
+            explorerUrl={getExplorerInfo(to.value)}
+            shortenHash={8}
+          />
+        </TxParameterWrapper>
+      )}
+
+      {safeTxHash && (
```

Comment:
Should we also move the `safeTxHash` up to the `safe nonce` and `safeTxGas`?

Reply:
I based it on the way `TxSummary` displayts data. I guess it's a question more suited for @johannesmoormann.